website/versioned_docs/version-29.0/JestObjectAPI.md

```diff
@@ -5,6 +5,10 @@ title: The Jest Object
 
 The `jest` object is automatically in scope within every test file. The methods in the `jest` object help create mocks and let you control Jest's overall behavior. It can also be imported explicitly by via `import {jest} from '@jest/globals'`.
 
+import TypeScriptExamplesNote from './_TypeScriptExamplesNote.md';
+
+<TypeScriptExamplesNote />
```

Comment:
Here it is about `jest.mocked` and `jest.Mocked`. Just in case, because these two currently differ from `@types/jest`.